backend/pvm_hmc.pm

```diff
@@ -0,0 +1,103 @@
+# Copyright © 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package backend::pvm_hmc;
+
+use strict;
+use warnings;
+
+use base 'backend::virt';
+
+use testapi qw(get_var get_required_var check_var);
```

Comment:
`check_var` is not used here.

Reply:
Also in general we should prefer to not use the external API anywhere inside. We can simply use `%bmwqemu::vars` directly

Reply:
I'd leave this decision up to the follow-up PR which is supposed to unify these backends if you're fine with this? (I honestly have no clue what you mean with `%bmwqemu::vars`)

Reply:
ok, I am only asking you for one thing then: Remind me of this after this PR here is done :)